scripts/build.sh

```diff
@@ -41,10 +41,17 @@ build_metadata() {
     if [ -z "${GITHUB_EVENT_NAME}" ]; then
         GITHUB_EVENT_NAME="manual"
     fi
+    if [ -z "${GITHUB_REF}" ]; then
+        GITHUB_REF="$(git show-ref HEAD)"
+    fi
+    if [ -z "${GITHUB_SHA}" ]; then
+        GITHUB_SHA="$(git show-ref -s HEAD)"
+    fi
+    GITHUB_BUILT="$(date -u -Iseconds)"
     jq -n \
-        --arg built "$(date -u -Iseconds)" \
-        --arg ref "$(git show-ref HEAD | awk '{print $2}')" \
-        --arg sha "$(git show-ref -s HEAD)" \
+        --arg built "${GITHUB_BUILT}" \
+        --arg ref "${GITHUB_REF##*/}" \
```

Comment:
What's the `##*/`?

Reply:
It removes the sha to keep only for example "fbw"